### DISCUSSION TITLE
axis 211a ip cam horizontal lines when low light

### DISCUSSION THREAD

**Discussion by neuronetv** [AxisCommunications/discussions#129] [General]
as the subject says, I've got image corruption in the form of flickering horizontal lines in the camera image during low light. see image attached.
is there anything I can do about this? axis model 211a
![211a-image](https://user-images.githubusercontent.com/10003170/204853753-5d24fabb-c5c4-414a-80b3-de180be40ee6.jpg)

**Comment by mattias-kindborg-at-work** [comment]
Questions here in GitHub Discussions are aimed for development discussions. See #13 for more information on how to contact Axis for product related questions.
